week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -7,8 +7,19 @@
   NOTE: only the names should be returned, not the means of transport.
 */
 
-function journeyPlanner() {
+function journeyPlanner(londonLocations, transport) {
+  let famousLocations = [];
 
+  for (let i = 0; i < londonLocations.length; i++) {
```

Comment:
Again good work, but we could use an array method here instead of a `for` loop. How could we change the code so it uses an array method?

Reply:
`function journeyPlanner(londonLocations, transport) {
  return  londonLocations.filter(famousLocation => famousLocation.includes(transport)).map(Location=>Location[0])
 }`

Reply:
👏